### DISCUSSION TITLE
How to add images in the webpage

### DISCUSSION THREAD

**Discussion by qwezxcmp130743** [colinwilson/lotusdocs#140] [Q&A]
I tried `![xxx](/static/images/xxx.webp)`
but it didn't reflect, can someone help me with it?
expecting the syntax, where to add and how to add
thanks in advance

**Comment by seanlowe** [comment]
I was running into the same issue but solved it via moving my images into `assets/images/` (create the `images/` directory if needed) and then use them in my markdown like so:
```markdown
![blah](/images/picture_name.whatever)
```
